tests/tensor/test_basic.py

```diff
@@ -4310,3 +4312,98 @@ def test_oriented_stack_functions(func):
 
     with pytest.raises(ValueError):
         func(a, a)
+
+
+class TestConvolve:
+    def setup_method(self):
+        self.op = convolve
+        self.op_class = Convolve
+
+    def _validate_output(self, a, v, mode):
+        aesara_sol = self.op(a, v, mode=mode).eval()
+        numpy_sol = np.convolve(a, v, mode=mode)
+        assert np.allclose(numpy_sol, aesara_sol)
```

Comment:
If this is only used in one place, we shouldn't need a method.

Reply:
It's used in all the tester methods.